### PR TITLE
Update changelog since v2.2.1

### PR DESCRIPTION
**Pull request by thomashoneyman**
**Description of the change**

This PR updates the CHANGELOG in preparation for the upcoming release of this library.

Related: purescript/purescript#3985

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,26 +5,28 @@ Notable changes to this project are documented in this file. The format is based
...
 
 ## [v2.2.0](https://github.com/purescript-contrib/purescript-strings-extra/releases/tag/v2.2.0) - 2020-09-13
 
-* Expose `upperCaseFirst` function
+- Exposed `upperCaseFirst` function
 
 ## [v2.1.0](https://github.com/purescript-contrib/purescript-strings-extra/releases/tag/v2.1.0) - 2020-04-08
 
-Add Levenshtein distance and Sorensen-Dice coefficient functions (@flip111)
+- Addde Levenshtein distance and Sorensen-Dice coefficient functions (@flip111)
```
Comment (toastal) [inline CHANGELOG.md]: Added

Comment (JordanMartinez) [reply CHANGELOG.md]: Fixed. Thanks!

**Comment by thomashoneyman** [comment]
The CI failures are a bit odd:

```
[1/2 UnknownImport] .spago/prelude/master/src/Data/Show.purs:8:23

  8  import Record.Unsafe (unsafeGet)
                           ^^^^^^^^^
  
  Cannot import value unsafeGet from module Record.Unsafe
  It either does not exist or the module does not export it.

[2/2 UnknownImport] .spago/safe-coerce/master/src/Safe/Coerce.purs:7:23

  7  import Unsafe.Coerce (unsafeCoerce)
                           ^^^^^^^^^^^^
  
  Cannot import value unsafeCoerce from module Unsafe.Coerce
  It either does not exist or the module does not export it.
```

I'm going to merge this as they're unrelated.

**Comment by JordanMartinez** [comment]
Is this caused by a caching issue?

**Pull request merged**